src/client/Microsoft.Identity.Client/TokenCache.Notifications.cs

```diff
@@ -3,6 +3,7 @@
 
 using System;
 using System.Threading.Tasks;
+using Microsoft.Identity.Client.PlatformsCommon.Factories;
```

Comment:
Is this needed? Seems like nothing here is referencing the new TokenCacheExtensions file?